### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by inkade**
Bulked out README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,24 +2,67 @@
...
+1. Clone the repository to your local machine and navigate to it:
+ ```bash
+> git clone --recurse-submodules https://github.com/webmachinelearning/webnn-samples
+> cd webnn-samples & npm install
+> npm start
+```
+2. Navigate to the desired sample directory that you want to explore.
+3. Read the accompanying README.md file for the sample to understand its purpose, requirements, and implementation details.
+4. Follow the instructions provided in the README to set up the necessary dependencies and run the sample.
+5. Experiment with the code and explore how WebNN can enhance machine learning tasks in the browser, navigating to http://localhost:8080.
```
Comment (Honry) [inline README.md:32]: We may need to emphase currently WebNN implementation only supports CPU backend on Chrome or Edge behind a flag: [#enable-experimental-web-platform-features](chrome://flags/#enable-experimental-web-platform-features)

**Review by Honry — COMMENTED** [review]
@inkade, awesome, thanks much for your contribution! That's really helpful!

**Comment by Honry** [comment]
@ibelem, PTAL as well. Thanks!

File: README.md (hunk 1 above)
Comment (ibelem) [reply README.md:32]: @Honry, I think @inkade may have no idea on this, we can add it by ourselves.

File: README.md [hunk 2]
```diff
@@ -2,24 +2,67 @@
...
+
+### WebNN API Samples
 * [WebNN code editor](https://webmachinelearning.github.io/webnn-samples/code/)
 * [Handwritten digits classification](https://webmachinelearning.github.io/webnn-samples/lenet/)
 * Noise suppression:
   * [NSNet2](https://webmachinelearning.github.io/webnn-samples/nsnet2/)
   * [RNNoise](https://webmachinelearning.github.io/webnn-samples/rnnoise/)
 * [Fast style transfer](https://webmachinelearning.github.io/webnn-samples/style_transfer/)
-* [Image classification](https://webmachinelearning.github.io/webnn-samples/image_classification/)
-* [Object detection](https://webmachinelearning.github.io/webnn-samples/object_detection/)
```
Comment (Honry) [inline README.md:13]: Mistake? Don't remove these two lines.

**Comment by inkade** [comment]
Thanks for the feedback guys, I appreciate it.

I have updated with the following:
* Switched to relative paths for 'Repository Structure' URLs.
* Emphasised the compatible browser requirement by placing the 'Requirements' section before 'Getting Started'.
* Provided clarity and guidance regarding enabling the #enable-experimental-web-platform-features flag.
* Reintergrated 'Image classification' and 'Object detection' links within 'WebNN API Samples'.

Regards

**Review by Honry — APPROVED** [review]
Pretty good! Thanks!

**Pull request merged**